src/main/java/com/redhat/sast/api/model/JobBatchDataFlow.java

```diff
@@ -9,25 +10,21 @@
 
 @Entity
 @Table(
-        name = "job_batch_data_lineage",
-        indexes = {
-            @Index(name = "idx_job_batch_data_lineage_lineage_id", columnList = "lineage_id"),
-            @Index(name = "idx_job_batch_data_lineage_transformation_type", columnList = "transformation_type"),
-            @Index(name = "idx_job_batch_data_lineage_created_at", columnList = "created_at")
-        })
+        name = "job_batch_data_flow",
+        indexes = {@Index(name = "idx_job_batch_data_flow_id", columnList = "id")})
 @Data
 @NoArgsConstructor
 @EqualsAndHashCode(exclude = "jobBatch")
-public class JobBatchDataLineage {
+public class JobBatchDataFlow {
 
     @Id
     private Long id; // Shared primary key with JobBatch
 
-    @Column(name = "lineage_id", unique = true, nullable = false)
-    private String lineageId;
+    @Column(name = "flow_id", unique = true, nullable = false)
+    private String flowId;
 
     @Column(name = "input_artifacts", columnDefinition = "jsonb", nullable = false)
-    private String inputArtifacts;
+    private List<String> inputArtifacts;
```

Comment:
Can we be a bit expressive on the variable name? It's not clear what we meant by `inputArtifacts`. Are we referring to the `inputArtifactUrls`?